browser/package.json

```diff
@@ -12,6 +12,19 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "parcel": "^2.6.2"
+    "@babel/preset-env": "^7.18.6",
+    "@babel/preset-react": "^7.18.6",
+    "babel-jest": "^28.1.2",
+    "jest": "^28.1.2",
+    "parcel": "^2.6.2",
+    "process": "^0.11.10",
+    "react-test-renderer": "^18.2.0"
+  },
+  "dependencies": {
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-icons": "^4.4.0",
```

Comment:
Why do we need this dependency?

Reply:
Which one?

Reply:
`react-icons` (which the commented line is targeting)